OracleWebCenterPortal/dockerfiles/12.2.1.4/README.md

```diff
@@ -76,7 +111,7 @@ The docker run command creates the container as well as starts the Admin Server
 * Node Manager
 * Admin Server
 
-When the command is run for the first time, we need to create the domain and configure the portal server, so following are done in sequence:
+When the command is run for the first time, the domain is created and the Portal Server is configured, so following are done in sequence
```

Comment:
Can you please add "Oracle WebCenter" before "Portal Server". You're also missing the word "the", i.e. "so _the_ following [tasks] are in done in sequence:"